### ISSUE TITLE
Curve tool error

### ISSUE THREAD

**Issue by rayluc3d** [aresdevo/animaide#47]
Hi there!

Running into this issue with the new 1.03 update when using any of the sliders. Currently using Blender 2.90.1

Traceback (most recent call last):
  File "E:\Work\StudioGooBot\GooScripts\_Addons\addons\animaide\curve_tools\ops.py", line 202, in invoke
    support.get_globals(context)
  File "E:\Work\StudioGooBot\GooScripts\_Addons\addons\animaide\curve_tools\support.py", line 191, in get_globals
    co = {'x': key.co_ui.x, 'y': key.co_ui.y}
AttributeError: 'Keyframe' object has no attribute 'co_ui'

location: <unknown location>:-1

Here are my animaide preferences: 
![image](https://user-images.githubusercontent.com/48029605/114935783-770d3c00-9df0-11eb-9126-a13f07a7806e.png)

**Comment by joaorafaeu** [comment]
Same problem here. Blender 2.92.0 Alpha

**Comment by aresdevo** [comment]
Sorry @rayluc3d, didn't realize this was still a thing in 2.90, I just tried it on that version and you are right. I'm working on a fix for this for people that are using the LTE version (2.83) that should work for 2.90 too. @joaorafaeu It is very strange this error is happening to you using version 2.92, I just tried it on version 2.91  and 2.92 and it works fine.

You can wait until I update the LTE version of the addon or download the latest stable Blender version. In the meantime, if you are not able to update to the latest Blender version you can download the LTE version of the addon here: 
https://github.com/aresdevo/animaide/archive/refs/heads/LTE.zip
I think that is the first version of the addon.

**Comment by rayluc3d** [comment]
Hey again! So I've updated to 2.93 Beta and I'm currently running into this error now:

Python: Traceback (most recent call last):
  File "C:\Users\raymo\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\animaide-master\curve_tools\ops.py", line 202, in invoke
    support.get_globals(context)
  File "C:\Users\raymo\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\animaide-master\curve_tools\support.py", line 173, in get_globals
    if not utils.curve.valid_fcurve(context, obj, fcurve):
  File "C:\Users\raymo\AppData\Roaming\Blender Foundation\Blender\2.93\scripts\addons\animaide-master\utils\curve.py", line 337, in valid_fcurve
    bone_name = split_data_path[1]
IndexError: list index out of range

location: <unknown location>:-1

Any help is greatly appreciated, thanks!

**Comment by aresdevo** [comment]
Sorry for the delay... I'll check that

**Comment by aresdevo** [comment]
Sorry for the delay, not been coding lately... life got in the way. 
I could not reproduce this. Could you please provide a bit of context on when this error happens?
if you could also send a Blender file it would help me a lot too.

Thanks for reporting this bug.

**Comment by aresdevo** [comment]
@rayluc3d  Sorry it took me so long to finally check this bug again. the issue you mention is something changed in the latest Blender version and I realized I don't have the bandwidth to support old versions. What I decided to do from now on is to have releases so the old versions of the addon are always available, but the issue you mentioned for Blender 2.9 is not something that will be addressed.

In regard to your second issue is an interesting find, I'm looking into it.
